auto_nag/utils.py

```diff
@@ -495,46 +505,17 @@ def get_human_lag(date):
     return humanize.naturaldelta(today - dt)
 
 
-def get_version(jsonVersion, key):
-    # In X.Y, we just need X
-    version = jsonVersion[key].split('.')
-    return version[0]
-
-
-def get_current_versions():
-    global _CURRENT_VERSIONS
-    if _CURRENT_VERSIONS is not None:
-        return _CURRENT_VERSIONS
-
-    jsonContent = requests.get(
-        'https://product-details.mozilla.org/1.0/firefox_versions.json'
-    ).json()
-    esr_next_version = get_version(jsonContent, 'FIREFOX_ESR_NEXT')
-    if esr_next_version:
-        esr_version = esr_next_version
-    else:
-        # We are in a cycle where we don't have esr_next
-        # For example, with 52.6, esr_next doesn't exist
-        # But it will exist, once 60 is released
-        # esr_next will be 60
-        esr_version = get_version(jsonContent, 'FIREFOX_ESR')
-
-    _CURRENT_VERSIONS = {
-        'central': get_version(jsonContent, 'FIREFOX_NIGHTLY'),
-        'beta': get_version(jsonContent, 'LATEST_FIREFOX_DEVEL_VERSION'),
-        'esr': esr_version,
-        'release': get_version(jsonContent, 'LATEST_FIREFOX_VERSION'),
-    }
-    return _CURRENT_VERSIONS
-
+def get_nightly_version_from_bz():
+    def bug_handler(bug, data):
+        status = 'cf_status_firefox'
+        N = len(status)
+        for k in bug.keys():
+            if k.startswith(status):
+                k = k[N:]
```

Comment:
Nit: remove N variable and just do `k = k[len(status):]`

Reply:
yeah, it's just to avoid a conflict between black and flake8